packages/office-addin-project/src/convert.ts

```diff
@@ -68,6 +70,10 @@ async function backupProject(backupPath: string) {
   });
 }
 
+export function convertManifest(manifestPath: string, outputPath: string) {
+  execSync(`node ./converter/lib/main.js --input ${manifestPath} --output ${outputPath} --verbose false --debug true`);
```

Comment:
I don't see this converter code.

Reply:
The conversion code should be in a separate package which is a dependency, and then "npx <package name>" can invoke the CLI for the package.  

You can also have it run an NPM script, which allows the script to have different commands if needed. An example of this is "npx run dev-server" where the "dev-server" script does the appropriate thing, but it isn't tightly coupled.

Reply:
Even when being developed locally, you can use "npm link" to have the package as a dependency.